starknet_py/hash/address.py

```diff
@@ -0,0 +1,35 @@
+from typing import Sequence
```

Comment:
Do you think having two `address` files is ok?

Reply:
As they are in different places and contain other logic, I'd say yes